webpack.config.js

```diff
@@ -45,5 +47,24 @@ module.exports = function (env = {}) {
     }
   }
 
+  if (env.production) {
+    config.entry['zaf_sdk.min'] = [
+      'native-promise-only',
+      './lib/index.js'
+    ]
+
+    config.module.rules.push({
+      test: /\.js$/,
+      use: { loader: 'babel-loader', options: { plugins: [], presets: ['babel-preset-env'] } }
```

Comment:
Should we use babel for development as well, for example, to test with various browser/os?